MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/service/database/SentimentsDatabase.java

```diff
@@ -0,0 +1,43 @@
+package com.google.moviestvsentiments.service.database;
+
+import android.content.Context;
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
+import com.google.moviestvsentiments.model.Account;
+import com.google.moviestvsentiments.service.account.AccountDao;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * SentimentsDatabase provides a high-level interface for performing database queries.
+ */
+@Database(entities = {Account.class}, version = 1, exportSchema = false)
+public abstract class SentimentsDatabase extends RoomDatabase {
+
+    /**
+     * accountDao returns an object that allows queries on the accounts table.
+     */
+    public abstract AccountDao accountDao();
```

Comment:
nit: please consider reordering fields and methods for readability. (https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents). (https://drive.google.com/file/d/1GEYGBrQpvZTo-ZZLB5hDwVIls0K8AHnz/view?usp=sharing)